packages/table/src/interactions/selectable.tsx

```diff
@@ -93,6 +101,27 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
             return false;
         }
 
+        const regionCardinality = Regions.getRegionCardinality(region);
+        let focusCellCoordinates: IFocusedCellCoordinates = null;
+
+        if (regionCardinality === RegionCardinality.FULL_TABLE) {
+            focusCellCoordinates = { col: 0, row: 0 };
+        }
+
+        if (regionCardinality === RegionCardinality.FULL_COLUMNS) {
+            focusCellCoordinates = { col: region.cols[0], row: 0 };
+        }
+
+        if (regionCardinality === RegionCardinality.FULL_ROWS) {
+            focusCellCoordinates = { col: 0, row: region.rows[0] };
+        }
+
+        if (regionCardinality === RegionCardinality.CELLS) {
+            focusCellCoordinates = { col: region.cols[0], row: region.rows[0] };
+        }
```

Comment:
Can we do `if/else-if` for all of these? Doesn't have the nice whitespace between, but makes it clearer that these are all disjoint cases.